x/ccv/provider/keeper/proposal.go

```diff
@@ -153,7 +153,7 @@ func (k Keeper) StopConsumerChain(ctx sdk.Context, chainID string, lockUbd, clos
 
 	k.DeleteInitChainHeight(ctx, chainID)
 	k.ConsumeSlashAcks(ctx, chainID)
```

Comment:
Nit (you can ignore it because it's out of scope): Maybe rename `ConsumeSlashAcks` to `DeleteSlashAcks` to stay consistent.

Reply:
Let's not add more changes than necessary. We can have these type of changes as a refactoring PR.